src/sentry/processing/realtime_metrics/__init__.py

```diff
@@ -6,17 +6,17 @@
 
 from .base import RealtimeMetricsStore
 
-realtime_metrics_store: RealtimeMetricsStore = LazyServiceWrapper(
```

Comment:
This wasn't the right type before and caused some confusion for mypy. Moved this type down the the `TYPE_CHECKING` area.

Reply:
oh, thanks for figuring this out.  i remember being confused why mypy was still not handling this.